code/game/objects/items/wayfinding.dm

```diff
@@ -1,121 +1,231 @@
-	var/interact_cooldown = 20 SECONDS //time per person for subsequent interactions
-	var/start_bal = 200 //how much money it starts with to cover wayfinder refunds
-	var/refund_amt = 40 //how much money recycling a pinpointer rewards you
+	///How many credits the dispenser account starts with to cover wayfinder refunds.
+	var/start_bal = 400
+	///How many credits recycling a pinpointer rewards you.
+	var/refund_amt = 40
 	var/datum/bank_account/synth_acc = new /datum/bank_account/remote
-	var/ppt_cost = 65 //Jan 6 '20: Assistant can buy one roundstart (125 cr starting)
+	var/ppt_cost = 0 //Jan 9 '21: 2560 had its difficulties for NT as well
 	var/expression_timer
+	///Avoid being Reddit.
+	var/funnyprob = 2
+	///List of slogans used by the dispenser to attract customers.
+	var/list/slogan_list = list("Find a wayfinding pinpointer? Give it to me. Please. I'll make it worth your while. Daddy needs his medicine.", //last sentence is a reference to Sealab 2021
+								"See a wayfinding pinpointer? Don't let it go to the crusher! Feed it to me instead! Please. I'll pay you.", //I see these things heading for disposals through cargo all the time
+								"Need the disk? Can't get a proper pinpointer? Buy a wayfinding pinpointer and find the captain's office today!",
+								"Bleeding to death? Can't read? Find your way to medbay today!", //there are signs that point to medbay but you need basic literacy to get the most out of them
+								"Voted tenth best pinpointer in the universe in 2560!", //there were no more than ten pinpointers in the game in 2020
+								"Helping assistants find the departments they tide since 2560.", //not really but it's advertising
+								"These pinpointers are flying out the airlock!", //because they're being thrown into space
+								"Grey pinpointers for the grey tide!", //I didn't pick the colour but it works
+								"Feeling lost? Find direction.",
+								"Automate your sense of direction. Buy a wayfinding pinpointer today!",
+								"Feed me a stray pinpointer.", //American Psycho reference
+								"We need a slogan!") //Liberal Crime Squad reference
+	///Number of the list entry of the slogan we last said.
+	var/previous_slogan_entry = 0
+	///Last world tick we said a slogan.
+	var/last_slogan_time = 0
 
 /obj/machinery/pinpointer_dispenser/Initialize(mapload)
 	. = ..()
-	var/datum/bank_account/civ_acc = SSeconomy.get_dep_account(ACCOUNT_CIV)
+	var/datum/bank_account/civ_acc = SSeconomy.get_dep_account(payment_department)
 	if(civ_acc)
 		synth_acc.transfer_money(civ_acc, start_bal) //float has to come from somewhere, right?
 
 	synth_acc.account_holder = name
 
-	desc += " Only [ppt_cost] credits! It also likes making costumes..."
+	desc += " [ppt_cost ? "Only [ppt_cost] credits! " : ""]It also synthesises costumes for some reason."
 
-	set_expression("neutral")
+	power_change()
+
+	last_slogan_time = world.time + rand(0, COOLDOWN_SLOGAN)
```

Comment:
```suggestion
	COOLDOWN_START(src, slogan_cooldown, COOLDOWN_SLOGAN)
```
Make a slogan cooldown min and max define if you are going to use `rand()`
And make the min value a bit higher than zero.